docs/reference/sql/limitations.asciidoc

```diff
@@ -113,3 +113,28 @@ FROM (SELECT ...) WHERE [simple_condition]`, this is currently **un-supported**.
 Using `FIRST` and `LAST` in the `HAVING` clause is not supported. The same applies to
 <<sql-functions-aggs-min,`MIN`>> and <<sql-functions-aggs-max,`MAX`>> when their target column
 is of type <<keyword, `keyword`>> as they are internally translated to `FIRST` and `LAST`.
+
+[float]
+=== Using TIME data type in GROUP BY or <<sql-functions-grouping-histogram>>
+
+Using `TIME` data type as a grouping key is currently not supported. For example:
+
+[source, sql]
```

Comment:
An alternative is to move these tests back to the docs.csv-spec and mark them as `-Ignore` which means they will not be ran.